packages/auth-express/tsconfig.json

```diff
@@ -1,13 +1,11 @@
 {
   "compilerOptions": {
-    "outDir": "./dist",
```

Comment:
Should this also extend `@repo/tsconfig/client.json`?

Reply:
hm, I tried to keep all old rules the same... this package only has these rules, client.json and base.json have more rules... but also, I can go again through all rules and see what are the best for our packages

Reply:
Yeah, I don't think I intended them to be wildly different, but maybe it makes sense to use the settings that are most popular with the framework's own ecosystem. I don't think there is probably a good candidate for Express like there is for the more full-featured meta-frameworks like Next and Remix, but if it can use some shared config like this, seems fine. In the end I don't think it matters much since consumers will be consuming the emitted `.js`+`.d.ts` files rather than the `.ts` files directly, so as long as we emit reasonable JS code, everything should be good.

The real reason to prefer extending one of our existing configs is just to make it easier/simpler on us to update these when needed.